maldoca/base/file.cc

```diff
@@ -18,8 +18,9 @@
 
 #include "maldoca/base/file.h"
 
-// TODO(someone): fix for windows
```

Comment:
Is there a github issue for this?

Reply:
no

Reply:
i am a little confused. this todo is removed as it is fixed (i.e. the code is not required for chrome)?

Reply:
Lol sorry, I was confused - I thought this was in green.